src/tls/extract/cstubs/mitlsffi.c

```diff
@@ -271,6 +271,21 @@ int MITLS_CALLCONV FFI_mitls_configure_alpn(/* in */ mitls_state *state, const c
     return 1;
 }
 
+int MITLS_CALLCONV FFI_mitls_configure_custom_extensions(/* in */ mitls_state *state, const mitls_extension *exts, size_t exts_len)
```

Comment:
Can you rename exts_len to exts_count?   I can see confusion if the caller passes sizeof(extension array) instead of # of elements in the array.